pkg/executor/grant.go

```diff
@@ -564,28 +573,46 @@ func (e *GrantExec) grantTableLevel(priv *ast.PrivElem, user *ast.UserSpec, inte
 	return err
 }
 
-// grantColumnLevel manipulates mysql.tables_priv table.
+// grantColumnLevel manipulates mysql.columns_priv and mysql.tables_priv table.
 func (e *GrantExec) grantColumnLevel(ctx context.Context, priv *ast.PrivElem, user *ast.UserSpec, internalSession sessionctx.Context) error {
 	dbName, tbl, err := getTargetSchemaAndTable(ctx, e.Ctx(), e.Level.DBName, e.Level.TableName, e.is)
 	if err != nil {
 		return err
 	}
+	ctx = kv.WithInternalSourceType(ctx, kv.InternalTxnPrivilege)
 
+	// 1. update `Column_priv` field in `mysql.tables_priv` table.
```

Comment:
Since the mysql.columns_priv and mysql.tables_priv may not consistance due to the history reason，can we check and make them consistance when user grant or revoke column related priviledges?

Reply:
Upgration will delete `column_priv` in `mysql.tables_priv` and `mysql.columns_priv`